app/components/create-session-message.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+
+  isMessageVisible  : true,
+  shouldShowMessage : computed('session.isAuthenticated', 'isMessageVisible', 'isNewSpeaker', 'isNewSession', function() {
+    let speakerIDlength = this.data.userSpeaker.toArray().length;
```

Comment:
what's this for, I don't get the logic?

Reply:
@CosmicCoder96 I think its to check if the speaker is added or not

Reply:
@CosmicCoder96 Just a moment, letting you where I picked it up from. Essentially it is for making sure that user has saved his/her speaker details in that particular cfs. If i must have opened any other event's cfs, I could get the popup already opened without any submission of speaker details. This basically avoids that faulty popup (see this : https://github.com/fossasia/open-event-frontend/blob/f35372d9cb5a208a93ee3fe3ceb125b9a1c42da0/app/templates/components/modals/cfs-proposal-modal.hbs#L34 )